zest/releaser/git.py

```diff
@@ -64,25 +64,25 @@ def tag_url(self, version):
         return version
 
     def cmd_diff(self):
-        return 'git diff'
+        return ['git', 'diff']
 
     def cmd_commit(self, message):
-        return 'git commit -a -m "%s"' % message
+        return ['git', 'commit', '-a', '-m', message]
 
     def cmd_diff_last_commit_against_tag(self, version):
-        return "git diff %s" % version
+        return ['git', 'diff', version]
 
     def cmd_log_since_tag(self, version):
         """Return log since a tagged version till the last commit of
         the working copy.
         """
-        return "git log %s..HEAD" % version
+        return ['git', 'log', '%s..HEAD' % version]
 
     def cmd_create_tag(self, version, sign=False):
         msg = "Tagging %s" % (version,)
-        cmd = 'git tag %s -m "%s"' % (version, msg)
+        cmd = ['git', 'tag', version, '-m', msg]
         if sign:
-            cmd += " --sign"
+            cmd.append["--sign"]
```

Comment:
I only noticed this one when trying to release it today:

      File "/Users/reinout/local/zest.releaser/zest/releaser/git.py", line 84, in cmd_create_tag
        cmd.append["--sign"]
    TypeError: 'builtin_function_or_method' object has no attribute '__getitem__'

:-) I'll fix it so that I can release zest.releaser again :-)